common/app/views/support/HtmlCleaner.scala

```diff
@@ -292,6 +292,21 @@ case class LiveBlogDateFormatter(isLiveBlog: Boolean)(implicit val request: Requ
   }
 }
 
+case class BloggerBylineImage(article: Article)(implicit val request: RequestHeader) extends HtmlCleaner  {
+  def clean(body: Document): Document = {
+    if (article.isLiveBlog && LiveBlogContributorImagesSwitch.isSwitchedOn) {
+      body.select(".block").foreach { el =>
+        val contributorId = el.attributes().get("data-block-contributor")
+        article.tags.find(_.id == contributorId).map{ contributorTag =>
```

Comment:
It looks like null is converted to an empty string in attributes.get. So maybe;
`val contributorId = el.attributes().get("data-block-contributor")`
`if (contributorId.nonEmpty) {`
`  ...` 
`}`